### ISSUE TITLE
UndockedRegFreeWinRT should support manifests with COM class activations in it

### ISSUE THREAD

**Issue by jlnordin** [microsoft/xlang#691]
I was testing out the code with a SxS manifest that included some COM activation information. This caused some of the WinRT entries to be corrupted, and I suspect a parsing error.

The RegFree code should look for WinRT activation entries but ignore any COM entries since COM activation is already supported down-level.

Sample manifest:

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
    <assemblyIdentity
        type="win32"
        name="TestWin32"
        version="1.0.0.0"
        />
    <dependency>
        <dependentAssembly>
            <assemblyIdentity type='win32' name='Microsoft.Windows.Common-Controls' version='6.0.0.0' processorArchitecture='*' publicKeyToken='6595b64144ccf1df' language='*'/>
        </dependentAssembly>
    </dependency>
    <file name="MyWinRTDll.dll">
        <activatableClass name="MyDll.MyComponent" threadingModel="both" xmlns="urn:schemas-microsoft-com:winrt.v1" />
    </file>
    <file name="MyWinRTDll2.dll">
        <activatableClass name="MyDll.MyComponent2" threadingModel="both" xmlns="urn:schemas-microsoft-com:winrt.v1" />
    </file>
    <file name="MyComDll.dll">
        <comClass
            description="MyDll_AClassicComClass"
            clsid="{2CBBF51B-75F7-4DA7-8703-D6BEC6817E1D}"
            threadingModel="Both"/>
    </file>
</assembly>
```

**Comment by jonwis** [comment]
What does "corrupted" mean in this sense? Like, the parse didn't work? Memory corruption in process? Missing or incorrect entries?

**Comment by jlnordin** [comment]
In this case "corrupted" means an invalid parse. It looks like other parts
of the xml file were included as the dll name field.

**Comment by kythant** [comment]
Now that I have taken a closer look, the parsing should just ignore the comClass tag. For the use of these classes, reg-free classic COM should handle these scenarios and the detours don't need to be involved at all. 
The parsing issue we saw was not because of these comClass tags. The parsing issues we saw with invalid file name should be fix and should work correctly now after this PR. 
https://github.com/microsoft/xlang/pull/692
